_guides/deploying-with-ksonnet.md

```diff
@@ -0,0 +1,519 @@
+---
+title: Deploying with KSonnet
+collection: How-to Guides
+layout: article
+section: On-prem
+category: component
+order: 1
+since: 20190119
+---
+
+One of the first problems this article solves is simplifying how users configure the applications they deploy to Kubernetes clusters.
+
+With that in mind, we are representing a detailed step by step instruction on how to implement the {{site.data.tenant.name}} platform's cluster and all the required Kubernetes descriptors onto a customer's hardware utilizing KSonnet.
+
+Before we go ahead and dive into the configuration peculiarities, let's deal with some of the basic terminologies.
+
+## What is Kubernetes 
```

Comment:
Why do we need this? We don't want to provide Kubernetes documentation. There is a better place for that. Please remove all the Kubernetes architecture stuff from here and just mention that the platform is running on top of Kubernetes and provide a link to the Kubernetes docs.

Reply:
As you wish. Consider it done.

P.S. According to one of our discussions (not reflected in the tech description, as always), this article should have also been well understood by users not familiar with Kubernets. That was just a brief intro to Kubernetes and not the "copy-paste from original Kurebnetes documentation."

Reply:
If you don't understand Kubernetes, this article is useless for you. Our platform runs on Kubernetes only and if you want to run the platform on your prem you must be a Kubernetes guru. Explaining to you what is Kubrnetes is a nonsense.